web-client/src/presenter/computeds/TrialSession/addTrialSessionInformationHelper.ts

```diff
@@ -31,12 +32,18 @@ export const addTrialSessionInformationHelper = (
     });
   }
 
+  // NOTE: what happens if isStandaloneSession is true and user is docketclerk?
```

Comment:
Unsure, of the business usecase here or if this state is even possible. Thought I'd mention it to get some feedback or context from the team.